install/kubernetes/istio-auth.yaml

```diff
@@ -1173,6 +1173,25 @@ spec:
         ports:
         - containerPort: 80
         - containerPort: 443
+        readinessProbe:
+          exec:
+            command:
+            - /bin/sh
+            - -c
+            - curl http://localhost:15000/clusters && curl http://localhost:15000/listeners && curl http://localhost:15000/routes
+          initialDelaySeconds: 3
```

Comment:
Thanks for putting this together. 

Are these numbers (seconds) default value? Just curious where we get this.

Reply:
@andraxylia we should use `curl -f ` so that curl actually fails on a non 200.